servicepulse/intro-archived-messages.md

```diff
@@ -4,28 +4,32 @@ summary: Describes how ServicePulse monitors archived failed messages, and allow
 component: ServicePulse
 tags:
  - ServicePulse
-reviewed: 2016-04-06
+reviewed: 2016-12-15
 ---
 
 
-In Versions 1.5.0 and above screens to manage archived messages have been introduced in response to changes in ServiceControl relating to Archived Message retention and disposal.
+In Versions 1.5.0 and above a screen to manage archived messages has been introduced in response to changes in ServiceControl relating to Archived Message retention and disposal.
 
-ServiceControl version 1.13.0 and above has a configurable [ErrorRetentionPeriod](/servicecontrol/creating-config-file.md) that schedules the disposal of archived failed messages.
-
-Archived messages can be found by going to the failed messages screen.
+Archived Messages can be found in a tab in the Failed Messages page. From the Failed Messages page, messages can also be archived from the Failed Groups tab, from the list of message corresponding to a certain Failed Group, and from the All Messages view.
 
-![Archived Messages Tab](images/archive.png 'width=500')
+Archiving in ServicePulse means that the failed messages are marked as "Archived". Data from an archived message is still available, but it is no longer displayed in the Failed Messages list in ServicePulse and is not counted by the Failed Messages indicator in the ServicePulse dashboard. It also will not appear in any failed message groups.
```

Comment:
That's from our perspective, but what does it mean from the user perspective? When will they use it?
E.g. Not all failed messages need to be retried, e.g. in IoT type of applications typically it's ok to miss a few measurements, and when something fails outdated data is not useful. If the failed messages don't need to be retried they can be archived, so they don't clutter the view.

Needs to be rephrases of course, but IMO that angle would be much more useful for the reader. Then we can add brief info about how does that appear, etc. The current paragraph looks good in that context.